components/common-go/experiments/flags.go

```diff
@@ -6,11 +6,15 @@ package experiments
 
 import "context"
 
+const (
+	PersonalAccessTokensEnabledFlag = "personalAccessTokensEnabled"
+)
+
 // IsMyFirstFeatureFlagEnabled example usage of a flag
 func IsMyFirstFeatureFlagEnabled(ctx context.Context, client Client, attributes Attributes) bool {
 	return client.GetBoolValue(ctx, "isMyFirstFeatureEnabled", false, attributes)
```

Comment:
Do you want a constant for this experiment name too?

Reply:
I'll remove that flag all together. Was there mostly as an example